organize/forms.py

```diff
@@ -0,0 +1,79 @@
+from django import forms
+from django_countries import countries
+from django_countries.fields import LazyTypedChoiceField
+from django_date_extensions.fields import ApproximateDateFormField
+
+
+from core.models import Event
+from core.validators import validate_approximatedate
+from .models import INVOLVEMENT_CHOICES
+
+PREVIOUS_ORGANIZER_CHOICES = (
+    (True, "Yes, I organized Django Girls"),
+    (False, "No, it’s my first time organizing Django Girls"))
+
+
+class PreviousEventForm(forms.Form):
+    has_organized_before = forms.ChoiceField(
+        widget=forms.RadioSelect,
+        choices=PREVIOUS_ORGANIZER_CHOICES)
+    previous_event = forms.ModelChoiceField(
+        queryset=Event.objects.past(), empty_label="Choose event",
+        required=False,
+        widget=forms.Select(
+            attrs={'aria-label': 'Choose event', 'class': 'linked-select'}))
+
+    def clean(self):
+        has_organized_before = self.cleaned_data.get('has_organized_before')
+        previous_event = self.cleaned_data.get('previous_event')
+        if (has_organized_before == 'True' and not previous_event):
+            raise forms.ValidationError({
```

Comment:
You can use `self.add_error('has_organized_before', "You have to choose an event.")` I think.